shared/src/test/factories.ts

```diff
@@ -248,6 +248,7 @@ export const generateIndustrySpecificData = (
     ]),
     employmentPersonnelServiceType: randomElementFromArray(["JOB_SEEKERS", "EMPLOYERS"]),
     employmentPlacementType: randomElementFromArray(["TEMPORARY", "PERMANENT", "BOTH"]),
```

Comment:
For testing only: if you want to validate the test. Will be removed before a merge